scripts/ci-docker-images/builder-x86_64-unknown-linux-gnu/Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN apt-get update && \
   curl \
   git \
   openssh-server \
-  vim
+  vim \
```

Comment:
we can probably remove this?

Reply:
We can, although not sure if we should if we are going to do #860.

Reply:
Actually, the `x86_64-unknown-linux-gnu` seems to already have CMake installed, but the version in it is too old.